src/main/java/com/minecolonies/coremod/generation/LootTableJsonProvider.java

```diff
@@ -0,0 +1,54 @@
+package com.minecolonies.coremod.generation;

```

Comment:
Should be part of the DataGen lib.
Or use the default one provided by Minecraft itself.

Reply:
This can't be in the DataGen lib since it depends on Minecraft types.